client/src/i18n/en.json

```diff
@@ -1,12 +1,26 @@
 {
   "ACCOUNT": {
+    "ACCOUNT_ASSET"    : "Asset Accounts",
```

Comment:
Why do you need to prefix all these keys with `ACCOUNT`?  In my opinion, `ACCOUNT.ASSET` is a much nicer key structure than `ACCOUNT.ACCOUNT_ASSET` (and lighter weight!).

This is a non-blocking comment.